src/elements/content-uploader/ItemAction.js

```diff
@@ -101,7 +101,12 @@ const ItemAction = ({
         <div className="bcu-item-action">
             {tooltip ? (
                 <Tooltip position="top-left" text={intl.formatMessage(tooltip)}>
-                    <PlainButton onClick={onClick} type="button" isDisabled={status === STATUS_STAGED}>
+                    <PlainButton
+                        onClick={onClick}
+                        type="button"
+                        isDisabled={status === STATUS_STAGED}
+                        aria-label={intl.formatMessage(tooltip)}
```

Comment:
nit: could we alphasort these props?

Reply:
yes, i need to think about things that way!

Reply:
don't worry bout it, we'll add a prettier config to handle this